pkg/apis/events/v1alpha1/pubsub_source_types.go

```diff
@@ -51,14 +51,15 @@ var _ = duck.VerifyType(&PubSubSource{}, &duckv1alpha1.Conditions{})
 
 // PubSubSourceSpec defines the desired state of the PubSubSource.
 type PubSubSourceSpec struct {
-	// GcpCredsSecret is the credential to use to poll the GCP PubSubSource Subscription. It is not used
+	// Secret is the credential to use to poll the PubSubSource Subscription. It is not used
```

Comment:
wonder if we should explicitly say which permissions this Service account should have?

Reply:
made an issue to make this documented better.